cs/src/comm/interfaces/Message.cs

```diff
@@ -5,47 +5,46 @@ namespace Bond.Comm
 {
     using System;
     using System.Diagnostics;
-    using System.Linq;
-    using System.Reflection;
 
     /// <summary>
     /// Interface representing a message of an unknown payload type.
     /// </summary>
-    /// <remarks>A message can contain either a payload or an error.</remarks>
+    /// <remarks>
+    /// A message can contain either a payload or an <see cref="Bond.Comm.Error"/>.
+    /// </remarks>
     public interface IMessage
     {
         /// <summary>
         /// Gets the payload as a <see cref="IBonded"/> value.
         /// </summary>
         /// <exception cref="InvalidOperationException">
-        /// Thrown when the message contains an error and not a payload.
+        /// Thrown when the message contains an Error and not a payload.
```

Comment:
Technically, lower-case error seems better than upper-case Error, due to the edge case where the payload is of type Error.